wdl/Manta.wdl

```diff
@@ -148,6 +151,12 @@ task RunManta {
       rm ./runWorkflow.py
     fi
 
+    if [ ! -z ~{select_first([bam_or_cram_index, ""])} ]; then
+      if [ ! -f `dirname ~{bam_or_cram_file}`/~{expected_index_name} ]; then
+        ln -s ~{bam_or_cram_index} `dirname ~{bam_or_cram_file}`/~{expected_index_name} || true
+      fi
+    fi
+
```

Comment:
Just a suggestion, I think this would be more readable:
```
String bam_ext = if is_bam then ".bam" else ".cram"
...
ln -s ~{bam_or_cram} sample.~{bam_ext}
ln -s ~{bam_or_cram_index} sample.~{bam_ext}.~{index_ext}
...
/usr/local/bin/manta/bin/configManta.py \
  --bam sample.~{bam_ext}
```

Reply:
One thing that I found tricky in this fix is that `bam_or_cram_index` is an optional input to this task. Do you know if there's some way to run it successfully without passing an index in that I don't understand?

Reply:
I see that it's always passed a value from 00a, so we can make it non-optional in the task and subworkflow.

Reply:
I liked your suggestion better than how I had it originally; changed to implement it.